core/bnd.bnd

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 ZXing authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Bundle-SymbolicName: com.google.zxing.core
+Import-Package: !com.google.zxing*
```

Comment:
Does it matter that core isn't the only artifact that exports packages in this namespace? (For example this is why the Java 9 module system won't work here).

Reply:
Not for this use case.

Reply:
Just wondering does other modules implement classes in the same packages as the core bundle?

Reply:
Yes, from years and years ago. `javase` has some classes in this package too.